packages/obojobo-document-engine/ObojoboDraft/Sections/Assessment/components/rubric/converter.js

```diff
@@ -0,0 +1,106 @@
+import ParameterNode from '../../../../../src/scripts/oboeditor/components/parameter-node'
+
+const RUBRIC_NODE = 'ObojoboDraft.Sections.Assessment.Rubric'
+const MOD_NODE = 'ObojoboDraft.Sections.Assessment.Rubric.Mod'
+const MOD_LIST_NODE = 'ObojoboDraft.Sections.Assessment.Rubric.ModList'
+
+const slateToObo = node => {
```

Comment:
Can these returns be converted similar to the comments I had made on the issue/next-166 branch (Return created object on last line rather than create empty object and add properties)